### ISSUE TITLE
panic: "index not found"

### ISSUE THREAD

**Issue by durka** [RustSec/cargo-audit#4]
`cargo audit` panics when run on my crate (separate panic from #3).

```
$ cargo install --debug cargo-audit
$ git clone git@github.com:durka/guard
Cloning into 'guard'...
remote: Counting objects: 221, done.
remote: Total 221 (delta 0), reused 0 (delta 0), pack-reused 221
Receiving objects: 100% (221/221), 30.36 KiB | 0 bytes/s, done.
Resolving deltas: 100% (121/121), done.
$ cd guard
$ cargo build
   Compiling guard v0.3.3
    Finished dev [unoptimized + debuginfo] target(s) in 0.30 secs
$ RUST_BACKTRACE=1 cargo audit
thread 'main' panicked at 'index not found', /Users/rustbuild/src/rust-buildbot/slave/beta-dist-rustc-mac/build/src/libcore/option.rs:715
stack backtrace:
   1:        0x107e5919c - std::sys::imp::backtrace::tracing::imp::write::h408ecc5a05e58f58
   2:        0x107e5adde - std::panicking::default_hook::{{closure}}::h726447131792b5be
   3:        0x107e5aa80 - std::panicking::default_hook::hfd15aba9d1568185
   4:        0x107e5b297 - std::panicking::rust_panic_with_hook::hccfc39a27e5d13b4
   5:        0x107e5b144 - std::panicking::begin_panic::hccbc74bfd8cc5618
   6:        0x107e5b062 - std::panicking::begin_panic_fmt::h72c157b5e14c69da
   7:        0x107e5afc7 - rust_begin_unwind
   8:        0x107e83280 - core::panicking::panic_fmt::hefe8a0512c8f5fb1
   9:        0x107e8330d - core::option::expect_failed::hcdbe6b8fc752fed3
  10:        0x107a04143 - <core::option::Option<T>>::expect::h8c58ee3681693373
  11:        0x107a130ae - <toml::value::Value as core::ops::Index<I>>::index::h4d178eeac1d00c04
  12:        0x107a1b823 - cargo_audit::load_lockfile::hf3e3b311d9bd0b93
  13:        0x107a1cb21 - cargo_audit::main::h5a22ca0922f89176
  14:        0x107e5c15a - __rust_maybe_catch_panic
  15:        0x107e5b666 - std::rt::lang_start::hf1490dffe05ecd2e
  16:        0x107a1d6c9 - main
```

**Comment by durka** [comment]
It looks like it just panics if the crate has no dependencies.

**Comment by tarcieri** [comment]
Fix released in cargo-audit 0.2.0
